statsmodels/discrete/tests/test_discrete.py

```diff
@@ -424,6 +424,7 @@ def setup_class(cls):
         res2.probit()
         cls.res2 = res2
         fit = Probit(data.endog, data.exog).fit
+        np.random.seed(1)
```

Comment:
Would anything be harmed by doing this for all CheckProbitSpector subclasses?  (Not that I'm suggesting doing that here, just want to understand)

Reply:
In general fitting a model with given dataset is deterministic (*). Only basinhopping needs a seed for the internal random number generation.
So, it was only necessary to add it to the basinhopping test

(*) up to floating point noise induced differences in convergence